app/javascript/crayons/MultiInput/MultiInput.jsx

```diff
@@ -14,6 +14,8 @@ const KEYS = {
  * Component allowing users to add multiple entries for a given input field that get displayed as destructive pills
  *
  * @param {Object} props
+ * @param {string} props.labelText The text for the input's label
```

Comment:
`labelText` and `showLabel` were introduced in [another in review PR](https://github.com/forem/forem/pull/18185) but has been added here as well as the test rely on it.